.env

```diff
@@ -1,4 +1,4 @@
-NEXT_PUBLIC_API_URL="https://ayushma-api.ohc.network/api/"
+NEXT_PUBLIC_API_URL="https://ayushma.ashsec.us.to/api/"
```

Comment:
@vishisht-dubey never change `.env` files. Almost always create a new `.env.local` file to overwrite during development

Reply:
ok sure